src/index.js

```diff
@@ -1,2 +1,59 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function renderTemplate(element) {
+  const app = document.getElementById('app');
+  app.textContent = '';
+  app.appendChild(element);
+}
+
+function render(count = 0) {
+  function handleClickNumber(value) {
+    render(value);
+  }
+
+  function handleClickCount() {
+    render(count + 1);
+  }
+
+  renderTemplate((
+    <div id="hello" className="greeting">
+      <p>Hello, world!</p>
+      <p>
+        <button type="button" onClick={() => handleClickCount()}>
```

Comment:
```suggestion
        <button type="button" onClick={handleClickCount}>
```

이렇게 작성하면 더 좋을 것 같아요. 이렇게 하면 `handleClickCount`에는 event객체가 매개변수로 전달됩니다

Reply:
@hannut91 감사합니다 :) event 객체로 전환하겠습니다.